diff --git a/visualization/fn_plot_point_verif.R b/visualization/fn_plot_point_verif.R
index 4ea9a8f..d5e6f7a 100644
--- a/visualization/fn_plot_point_verif.R
+++ b/visualization/fn_plot_point_verif.R
@@ -26,7 +26,8 @@
                                      cyc_summary = c("00","12","All"),
                                      clim_name = NULL,
                                      plot_num_obs = FALSE,
-                                     fcst = NULL){
+                                     fcst = NULL,
+                                     month = "All"){
  
   #=================================================#
   # INITIAL CHECKS
@@ -367,7 +368,7 @@
   }
   
   # Define the colour scheme used in line plots
-  mcolors <- fn_gen_model_colors(model_names,cmap)
+  mcolors <- fn_gen_model_colors(model_names,cmap,custom_colors = custom_colors)
   
   # Line stylces
   line_styles <- c("solid","dashed","dotted","dotdash")
@@ -432,6 +433,7 @@
                         "score_sep"    = score_sep, 
                         "comp_val"     = attributes(verif)$comp_val,
                         "thr_brks"     = attributes(verif)$thr_brks,
+                        "month"        = month,
                         "clim_name"    = clim_name)
   
   #=====================================================#
@@ -692,9 +694,9 @@
             num_stations <- ns_val
           }
           if (numbers_only(station)) {
-            subtitle_str <- paste0("Station ",station," (",num_stations,") : ",cy_str)
+            subtitle_str <- paste0("Month: ",month,", ","Station ",station," (",num_stations,") : ",cy_str)
           } else {
-            subtitle_str <- paste0(station," stations (",num_stations,") : ",cy_str)
+            subtitle_str <- paste0("Month: ",month,", ",station," stations (",num_stations,") : ",cy_str)
           }
           # Add in lead times used for valid_dttm
           if ((xgroup %in% c("valid_dttm","valid_ymd")) & (!is.null(alta))) {
