diff --git a/visualization/fn_plot_point_verif.R b/visualization/fn_plot_point_verif.R
index 01fbc7e..4ea9a8f 100644
--- a/visualization/fn_plot_point_verif.R
+++ b/visualization/fn_plot_point_verif.R
@@ -22,7 +22,8 @@ fn_plot_point_verif <- function(harp_verif_input,
                                      n_stations = NULL,
                                      use_parallel = FALSE,
                                      plot_num_obs = FALSE,
-                                     fcst = NULL){
+                                     fcst = NULL,
+                                     month = "All"){
  
   #=================================================#
   # INITIAL CHECKS
@@ -355,7 +356,7 @@ fn_plot_point_verif <- function(harp_verif_input,
   }
   
   # Define the colour scheme used in line plots
-  mcolors <- fn_gen_model_colors(model_names,cmap)
+  mcolors <- fn_gen_model_colors(model_names,cmap,custom_colors = custom_colors)
   
   # Line stylces
   line_styles <- c("solid","dashed","dotted","dotdash")
@@ -419,7 +420,8 @@ fn_plot_point_verif <- function(harp_verif_input,
                         "png_projname" = png_projname,
                         "score_sep"    = score_sep, 
                         "comp_val"     = attributes(verif)$comp_val,
-                        "thr_brks"     = attributes(verif)$thr_brks)
+                        "thr_brks"     = attributes(verif)$thr_brks,
+                        "month"        = month)
   
   #=====================================================#
   # NOW DO THE PLOTTING
@@ -663,9 +665,9 @@ fn_plot_point_verif <- function(harp_verif_input,
             num_stations <- ns_val
           }
           if (numbers_only(station)) {
-            subtitle_str <- paste0("Station ",station," (",num_stations,") : ",cy_str)
+            subtitle_str <- paste0("Month: ",month,", ","Station ",station," (",num_stations,") : ",cy_str)		  
           } else {
-            subtitle_str <- paste0(station," stations (",num_stations,") : ",cy_str)
+            subtitle_str <- paste0("Month: ",month,", ",station," stations (",num_stations,") : ",cy_str)		  
           }
           # Add in lead times used for valid_dttm
           if ((xgroup %in% c("valid_dttm")) & (!is.null(alta))) {
